app/Http/Controllers/Api/v1/EntityMoveApiController.php

```diff
@@ -39,13 +40,21 @@ public function transfer(Request $request, Campaign $campaign)
                     ->validate()
                     ->process()
                 ;
-                $count++;
+                    $count++;
+                }
             }
-        }
 
-        if ($copy) {
-            return response()->json(['success' => 'Succesfully copied ' . $count . ' entities.']);
+            if ($copy) {
+                return response()->json(['success' => 'Succesfully copied ' . $count . ' entities.']);
+            }
+            return response()->json(['success' => 'Succesfully transfered ' . $count . ' entities.']);
+        
+        } catch (Exception $e) {
```

Comment:
Needs a catch for TranslatableException, and for standard exception to be handled by the rest of the app. If it's a standard exception, I need to get an email that something went wrong with the code :)